src/components/ui/IconButton.tsx

```diff
@@ -0,0 +1,47 @@
+import React, {
+  ButtonHTMLAttributes,
+  PropsWithChildren,
+  forwardRef,
+  ForwardedRef,
+  useId,
+} from 'react'
+import { Button, Icon } from '@/components/index'
+import clsx from 'clsx'
+
+export interface IconButtonProps
+  extends ButtonHTMLAttributes<HTMLButtonElement> {
+  label: any
+}
+
+export const IconButton = forwardRef(
+  (
+    {
+      label,
+      id,
+      children,
+      type,
```

Comment:
wordt `type` gebruikt?

Reply:
Whoops, nee (was een left over van trying to figure out hoe ik de error kon oplossen 🙃). 

Heb 'm eruit gehaald!